include/model_card_manager.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef CARD_MANAGER_HPP_
+#define CARD_MANAGER_HPP_
+#include <memory>
+#include <string>
+#include <vector>
+#include "model_card.hpp"
```

Comment:
FYI: there are 2 different approaches for includes: 
1) include EVERYTHING that you are using despite the fact that this include is already presented in your other files
(e.g. you are using string in your translation unit, so you include `<string>` even though it is already included in your other header file which is already included in this translation unit)
2) include only what is not presented in other includes

First approach seems weird to me (just personal preferences), so I would suggest stick to second approach. In this case you will need to firstly include headers with "higher-level logic" and append headers with low-level logic in the very end if it is necessary. 
So in your case you should fix your include order the following way:
```
#include "model_card.hpp"
#include <memory>
#include <string>
#include <vector>
```